src/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Dockerfile
+# A blueprint for building a docker image
+FROM python:3.9
+
+ADD move_embargo_scratch.py .
+
+RUN pip install whatever_you_need
```

Comment:
This should generally take its dependencies from a `requirements.txt` file so this becomes `pip install -r requirements.txt`.  In this case, that file would include `lsst-daf-butler` and `astropy` (if that's actually necessary).

Reply:
I changed this line in the dockerfile, but still need to figure out how to auto generate the requirements.txt file

Reply:
I don't think there's a good way to auto-generate `requirements.txt`.  It's usually manually maintained.

Reply:
I've added a requirements.txt file, but I now have questions about how we know what to add here; aren't all of our python imports included in the rubin environment? Does this mean we need to pip freeze a requirements file in order to capture all the modules in the rubin environment since the program will eventually be running in a Docker environment? Or can I simply modify the Dockerfile to have a line that reads `RUN setup lsst_distrib` and then we'll automatically have access to the modules we need?